### PR TITLE
opt: move taskpool at endpoint layer

### PR DESCRIPTION
**Pull request by watermelo**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by wongoo** [comment]
using `go mod tidy` to remove unused lib version in go.sum

File: server.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"fmt"
+	gxsync "github.com/dubbogo/gost/sync"
```
Comment (AlexStocks) [inline server.go:25]: move out to 3rd import block.

**Comment by watermelo** [comment]
> using `go mod tidy` to remove unused lib version in go.sum

done

File: server.go (hunk 1 above)
Comment (watermelo) [reply server.go]: done

**Review by AlexStocks — APPROVED** [review]

**Review by fangyincheng — APPROVED** [review]

**Review by wongoo — APPROVED** [review]

**Comment by AlexStocks** [comment]
image

**Pull request merged**